cmd/metagenomics/shortReadMNGS.go

```diff
@@ -0,0 +1,106 @@
+package metagenomics
+
```

Comment:
Should the name of this file change as well?

Reply:
thanks for the callout good point